internal/configuration/validator/logging.go

```diff
@@ -0,0 +1,56 @@
+package validator
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/authelia/authelia/internal/configuration/schema"
+	"github.com/authelia/authelia/internal/utils"
+)
+
+// ValidateLogging validates the logging configuration.
+func ValidateLogging(configuration *schema.Configuration, validator *schema.StructValidator) {
+	applyDeprecatedLoggingConfiguration(configuration, validator)
+
+	if configuration.Logging.Level == "" {
+		configuration.Logging.Level = schema.DefaultLoggingConfiguration.Level
+	}
+
+	if configuration.Logging.Format == "" {
+		configuration.Logging.Format = schema.DefaultLoggingConfiguration.Format
+	}
+
+	if configuration.Logging.FilePath == "" && configuration.Logging.KeepStdout {
```

Comment:
This set of configuration (which I suspect would be incorrectly applied by a user) would result in no logging.

I think we can strip the second part of the condition here and if `FilePath = ""` we should `log.Fatal` and exit so the user can setup their logging appropriately.

Mainly because KeepStdout doesn't actually do anything unless the FilePath is not blank:

https://github.com/authelia/authelia/blob/4cfda7eeceec77799bb4ccc7b783940426413365/internal/logging/logger.go#L33-L53